app/Models/Classroom.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Classroom extends Model
+{
+    use HasFactory;
+
+    protected $table = 'classrooms';
+    protected $fillable = [
```

Comment:
please check your settings of phpStorm. Please use 'space' character to indent

Reply:
fixed